marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/provenance/provenanceWriteQueue.sjs

```diff
@@ -0,0 +1,120 @@
+/**
+ Copyright (c) 2021 MarkLogic Corporation
+
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+'use strict';
+const consts = require("/data-hub/5/impl/consts.sjs");
+const hubUtils = require("/data-hub/5/impl/hub-utils.sjs");
+const ps = require('/MarkLogic/provenance');
+
+const provenanceNamespaces = {
+  dhf:"http://marklogic.com/dhf",
+  job:"http://marklogic.com/data-hub/job#",
+  step:"http://marklogic.com/data-hub/step#",
+  user:"http://marklogic.com/data-hub/user#",
+  dh:"http://marklogic.com/data-hub/provenance#"
+};
+/**
+ * Captures the content objects that should be written to a database after one or more steps
```

Comment:
This should be updated to describe this class instead of the writeQueue.sjs class.

Reply:
Updated this comment